File: Java/DailyFlash/July/30July/Program5.java

import java.util.Scanner;

class GreaterLessEqual {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		System.out.print("Enter a number: ");
		int num= sc.nextInt();

		if(num>0) {

			System.out.println("Enter number is greater than Zero");
		}
		else if(num==0) {

			System.out.println("Enter number is equal to Zero");
		}
		else {

			System.out.println("Enter number is less than Zero");
		}
	}
}

File: PPA/DailyFlash/July/24July/Program5.c


#include<stdio.h>

void main() {

	for(int i=1; i<=5; i++) {
		int num = 5;;
		for(int j=4; j>=i; j--) {

			printf("  ");
			num--;
		}
		for(int k=1; k<=i; k++) {

			printf("%d ",num);
			num--;
		}
		printf("\n");
	}
}
